message_definitions/v1.0/ras_a.xml

```diff
@@ -10,7 +10,140 @@
+        <param index="3" label="Approach vector initial point Y or Longitude">Y (mE4) or Longitude (degE7) (WGS84) coordinate of the initial point of the approach vector to the portal. NaN or INT32_MAX  if unknown.</param>
+        <param index="4" label="Approach vector initial point Z or Altitude (MSL)" units="m">Z or Altitude (MSL) coordinate of the initial point of the approach vector to the portal. NaN if unknown.</param>
+        <param index="5" label="Approach vector final point X or Latitude">X (mE4) or Latitude (degE7) (WGS84) coordinate of the final point of the approach vector to the portal. NaN or INT32_MAX  if unknown.</param>
+        <param index="6" label="Approach vector final point Y or Longitude">Y (mE4) or Longitude (degE7) (WGS84) coordinate of the final point of the approach vector to the portal. NaN or INT32_MAX if unknown.</param>
+        <param index="7" label="Approach vector final point Z or Altitude (MSL)" units="m">Z or Altitude (MSL) coordinate of the final point of the approach vector to the portal. NaN if unknown.</param>
+      </entry>
+      <entry value="40502" name="MAV_CMD_SET_EGRESS_PORTAL" hasLocation="true" isDestination="false">
+        <wip/>
+        <!-- This command is work-in-progress and it can therefore change. It should NOT be used in stable production environments. -->
+        <description>
+          Sets a specific portal to be an entry portal. Defines also the approach vector for a vehicle to approach and go through the portal.
+          COMMAND_INT should be used so to set the MAV_FRAME and consequently, frame and coordinates of the approach vector (MAV_FRAME_GLOBAL means global coordinates are being used,
+          while MAV_FRAME_LOCAL_NED means local NED coordinates are being used).
+        </description>
+        <param index="1" label="Portal ID" minValue="0" maxValue="4294967295" increment="1">ID of the portal to be set as an egress point.</param>
+        <param index="2" label="Approach vector initial point X or Latitude">X (mE4) or Latitude (degE7) (WGS84) coordinate of the initial point of the approach vector to the portal. NaN or INT32_MAX  if unknown.</param>
+        <param index="3" label="Approach vector initial point Y or Longitude">Y (mE4) or Longitude (degE7) (WGS84) coordinate of the initial point of the approach vector to the portal. NaN or INT32_MAX  if unknown.</param>
+        <param index="4" label="Approach vector initial point Z or Altitude (MSL)" units="m">Z or Altitude (MSL) coordinate of the initial point of the approach vector to the portal. NaN if unknown.</param>
+        <param index="5" label="Approach vector final point X or Latitude">X (mE4) or Latitude (degE7) (WGS84) coordinate of the final point of the approach vector to the portal. NaN or INT32_MAX  if unknown.</param>
```

Comment:
You can't set INT32 if this sent in a COMMAND_INT. Ditto for param 6

Reply:
Yes I can. Eg. https://mavlink.io/en/messages/common.html#MAV_CMD_DO_ORBIT. I send a NaN for COMMAND_LONG, and INT32_MAX for COMMAND_LONG.

Reply:
But considering this is a COMMAND_INT (always), what I can't send is a NaN,